### DISCUSSION TITLE
Instance that was offline for a while now getting 401 errors on /inbox

### DISCUSSION THREAD

**Discussion by SteveDinn** [mastodon/mastodon#32102] [General]
Hey. I have a technical issue that I don't understand and I'm hoping that I can get somebody with some expertise to help me out. Here's the situation:
1. I have a personal Mastodon instance that has been running the whole time and is in good health.
2. I have a GoToSocial instance that I was testing out. I had it working and communicating properly with my Mastodon instance.
3. I took my GoToSocial instance offline, and left it that way for several months, but I recently brought it back online.
4. I found the GoToSocial domain in my Mastodon admin settings (`/admin/instances?availability=unavailable`) and restarted delivery.
5. Even after that, the instances didn't seem to want to talk. The GTS instance was getting 401 errors trying to push ActivityPub things to `/inbox` on my Mastodon instance
6. After that, I tried something that may have been a mistake, but I clicked the "Purge" button on that Unavailable domains page after it has failed again enough times to show up there. I was hoping to make my Mastodon instance forget that it had ever seen my GTS instance, but this didn't work.
7. Now I have no record of the GTS domain on my Mastodon instance, but the GTS instance is still getting 401 errors when POSTing to `/inbox` and I have no idea why.
8. No settings or configuration on the GTS instance were changed at all.

Curiously enough, other instances have been able to successfully restart communication with my GTS instance, but not my own Mastodon instance!

That's where I am today. Is my GTS instance just FUBAR'd, or is there a "forget everything you know about this foreign instance" button in my Mastodon admin settings that I haven't found yet?

**Comment by dlitz** [comment]
Did you ever solve this?  I'm having a similar issue.

**Comment by dlitz** [comment]
Never mind.  In in my case, it looks like the date/time on my server wasn't being synchronized by NTP and had drifted by a few minutes, causing signature failures and 401 errors from basically everyone.
